src/Controllers/SoundingBoardController.php

```
@@ -170,54 +200,73 @@ public function results($id): void
-            $this->response->json(['error' => 'Invalid member index'], 400);
-            return;
-        }
+        $user = $this->auth->user();
 
-        // Update the individual response status
-        $results[$memberIndex]['status'] = $action . 'ed';
-// Update the stored JSON
-        $this->db->query("UPDATE evaluation_results SET results_json = :results_json WHERE id = :id", [
+        $this->db->beginTransaction();
+        try {
+            $eval = EvaluationResult::findByIdForUpdate($this->db, (int) $id);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\EvaluationResult' in method 'respond'. (undefined)

(StaticAccess)